src/Core/View.js

```diff
@@ -250,9 +250,10 @@ function _preprocessLayer(view, layer, provider) {
  * @param {Layer=} parentLayer
  * @return {Promise} a promise resolved with the new layer object when it is fully initialized
  */
+
 View.prototype.addLayer = function addLayer(layer, parentLayer) {
     const duplicate = this.getLayers((l => l.id == layer.id));
-    if (duplicate.length > 0) {
+    if (duplicate.length > 0 && !parentLayer) {
```

Comment:
No need for a `clone` method IMHO.

When you want to reuse the same datasource you have 2 alternatives:
- do like `cubic_planar.js` demo (https://github.com/iTowns/itowns/blob/master/examples/cubic_planar.js#L113 : the same *layer* is attached to several geometry layer but with a different id each time (so 1 `Layer` is only attached once). 
Advantage:
    * it already works
    * an `id` identifies a specific Layer and its attach "point"  (ie: you can use this `id` anywhere in itowns and it will only affect one thing

- we can allow attaching the same `Layer`object multiple times.

  Advantages:
    * Shorter syntax

  Risks:
    * one `id` now refers to distinct visual objects (so modifying 1 will affect others)
    * the syntax might be a bit weird, since the original `addLayer` do not return the `Layer` object but only its `whenReady` promise.